### PR TITLE
chore(scalasdk): cleanup and implement scala customer registry example.

### PR DESCRIPTION
**Pull request by RayRoestenburg**

### CODE REVIEW

File: codegen/scala-gen/src/main/scala/com/akkaserverless/codegen/scalasdk/impl/MainSourceGenerator.scala [hunk 1]
```diff
@@ -47,7 +47,7 @@ object MainSourceGenerator {
     }
```
Comment (patriknw) [inline codegen/scala-gen/src/main/scala/com/akkaserverless/codegen/scalasdk/impl/MainSourceGenerator.scala:47]: one more thing to enable there:
```
          case _: ModelBuilder.ValueEntity =>
            List(name -> service)
```

File: samples/scala-value-entity-customer-registry/src/main/scala/customer/action/customer_action/CustomerActionImpl.scala [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import com.akkaserverless.scalasdk.action.ActionCreationContext
+import com.google.protobuf.empty.Empty
+import customer.api.customer_api.Customer
+
+/** An action. */
+class CustomerActionImpl(creationContext: ActionCreationContext) extends AbstractCustomerAction {
+
+  /** Handler for "Create". */
+  override def create(customer: Customer): Action.Effect[Empty] = {
+    throw new RuntimeException("The command handler for `Create` is not implemented, yet")
```
Comment (johanandren) [inline samples/scala-value-entity-customer-registry/src/main/scala/customer/action/customer_action/CustomerActionImpl.scala:17]: Missing the forward in the Java one

Comment (RayRoestenburg) [reply samples/scala-value-entity-customer-registry/src/main/scala/customer/action/customer_action/CustomerActionImpl.scala:17]: Ah right thanks, must still implement the action

File: samples/scala-value-entity-customer-registry/src/main/resources/logback.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<configuration>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
```
Comment (patriknw) [inline samples/scala-value-entity-customer-registry/src/main/resources/logback.xml:2]: this should be changed to json, same as we have in the java samples, but could be done separately after #496

File: samples/scala-value-entity-customer-registry/src/main/scala/customer/domain/CustomerValueEntity.scala [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    effects.reply(convertToApi(currentState))
+
+  private def convertToDomain(customer: api.Customer) =
+    CustomerState(
+      customerId = customer.customerId,
+      email = customer.email,
+      name = customer.name,
+      address = customer.address.map(convertAddressToDomain))
+
+  private def convertAddressToDomain(address: api.Address) = {
```
Comment (patriknw) [inline samples/scala-value-entity-customer-registry/src/main/scala/customer/domain/CustomerValueEntity.scala:49]: superflous `{`

**Review by patriknw — APPROVED** [review]
LGTM, just something minor...

**Comment by patriknw** [comment]
@RayRoestenburg a conflict here

File: samples/scala-value-entity-customer-registry/src/main/scala/customer/domain/CustomerValueEntity.scala (hunk 4 above)
Comment (RayRoestenburg) [reply samples/scala-value-entity-customer-registry/src/main/scala/customer/domain/CustomerValueEntity.scala]: fixed

File: samples/scala-value-entity-customer-registry/src/main/resources/logback.xml (hunk 3 above)
Comment (RayRoestenburg) [reply samples/scala-value-entity-customer-registry/src/main/resources/logback.xml]: done

File: samples/scala-value-entity-customer-registry/src/main/scala/customer/action/customer_action/CustomerActionImpl.scala (hunk 2 above)
Comment (RayRoestenburg) [reply samples/scala-value-entity-customer-registry/src/main/scala/customer/action/customer_action/CustomerActionImpl.scala]: added

**Comment by RayRoestenburg** [comment]
Rebased on top of https://github.com/lightbend/akkaserverless-java-sdk/pull/499

**Review by johanandren — APPROVED** [review]
LGTM

**Pull request merged**